testing-pre-releases.md

```diff
@@ -0,0 +1,264 @@
+# Testing pre-release versions of Kubernetes with kubeadm
+
+## Overview
+
+When testing Kubernetes with kubeadm three different version numbers should be considered:
+
+- The version of .deb or .rpm packages, including kubelet, kubectl and related dependencies (e.g. kubernetes-cni);
+  it is convenient to include also kubeadm package in this group to get the kubelet drop-in file deployed automatically.
+- The version of the kubeadm binary to be used, that can eventually overwrite the kubeadm binary included in
+  the .deb or .rpm packages. e.g. Use v1.10.3 GA packages for kubelet, kubectl, but test a local build of kubernetes.
+- The `--kubernetes-version` flag of kubeadm or the `kubernetesVersion` field of Kubeadm API,
+  defining the version number of the control plane components installed by kubeadm.
+
+For each of the above set version number, it is also necessary to understand the supported version skew policy,
```

Comment:
What exactly does this look like? It's hard to suss out the actions you should be taking based on this.